### PR TITLE
feat(ci): Remove leading slash to ease future file parsing

### PR DESCRIPTION
**Pull request by chouetz**
### What does this PR do?
The leading slash is not necessary and would require additional manipulation in file path to read the included files

### Motivation
Simplify the code for future addition of method to generate full configuration

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

**Review by brycekahle — APPROVED** [review]

**Review by clarkb7 — APPROVED** [review]

**Review by derekwbrown — APPROVED** [review]
reviewed two wkit owned files

**Review by spencergilbert — APPROVED** [review]

File: .gitlab/benchmarks/include.yml [hunk 1]
```diff
@@ -3,5 +3,5 @@
 # Contains jobs to benchmark the Agent.
 
 include:
-  - /.gitlab/benchmarks/benchmarks.yml
-  - /.gitlab/benchmarks/macrobenchmarks.yml
\ No newline at end of file
+  - .gitlab/benchmarks/benchmarks.yml
+  - .gitlab/benchmarks/macrobenchmarks.yml
```
Comment (CelianR) [inline .gitlab/benchmarks/include.yml:7]: NIT: You can add a trailing line feed

File: .gitlab/internal_kubernetes_deploy/include.yml [hunk 2]
```diff
@@ -3,5 +3,5 @@
 # Contains jobs to trigger a pipeline in our k8s-datadog-agent-ops repo
 
 include:
-  - /.gitlab/internal_kubernetes_deploy/internal_kubernetes_deploy.yml
-  - /.gitlab/internal_kubernetes_deploy/rc_kubernetes_deploy.yml
\ No newline at end of file
+  - .gitlab/internal_kubernetes_deploy/internal_kubernetes_deploy.yml
+  - .gitlab/internal_kubernetes_deploy/rc_kubernetes_deploy.yml
```
Comment (CelianR) [inline .gitlab/internal_kubernetes_deploy/include.yml:7]: NIT: You can add a trailing line feed

File: .gitlab/kitchen_cleanup/include.yml [hunk 3]
```diff
@@ -2,5 +2,5 @@
 # kitchen_cleanup stage
 # Include file for jobs which clean up kitchen resources created for Agent kitchen tests.
 include:
-  - /.gitlab/kitchen_cleanup/cleanup.yml
-  - /.gitlab/kitchen_cleanup/kitchen_cleanup.yml
\ No newline at end of file
+  - .gitlab/kitchen_cleanup/cleanup.yml
+  - .gitlab/kitchen_cleanup/kitchen_cleanup.yml
```
Comment (CelianR) [inline .gitlab/kitchen_cleanup/include.yml:6]: NIT: You can add a trailing line feed

**Review by CelianR — APPROVED** [review]

**Review by L3n41c — APPROVED** [review]

**Review by pawelchcki — APPROVED** [review]

**Review by ajgajg1134 — APPROVED** [review]
looks good for agent-apm

**Pull request merged**